core/src/main/java/tc/oc/pgm/rotation/MapPoll.java

```diff
@@ -180,7 +180,7 @@ private Component getMapBookComponent(MatchPlayer viewer, MapInfo map) {
                 TextComponent.of(
                     map.getTags().stream().map(MapTag::toString).collect(Collectors.joining(" ")),
                     TextColor.YELLOW)))
-        .clickEvent(ClickEvent.runCommand("/votenext " + map.getName()))
+        .clickEvent(ClickEvent.runCommand("/votenext -f " + map.getName()))
```

Comment:
```suggestion
        .clickEvent(ClickEvent.runCommand("/votenext " + map.getName()))
```

Reply:
How about `-o` for open. I think we may want to keep the flag, to allow running the command without opening the book.